docs/components/sensor/_index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Sensor Component"
+linkTitle: "Sensor"
+weight: 70
+no_list: true
+type: "docs"
+description: "A device that sends information about the outside world to the computer controlling a robot."
+tags: ["sensor", "components"]
+icon: "img/components/sensor.png"
+# SME: #team-bucket
+---
+
+A *sensor* is a device that can measure information about the outside world.
+Add a sensor component to your robot to send the information the sensor measures to the computer controlling the robot.
+
+Most robots with a sensor need at least the following hardware:
+
+- A [board](/components/board/)
+- Depending on your sensor's output type (analog or digital), an analog-to-digital converter (ADC) may be necessary to allow the sensor to communicate with the board
+
+## Configuration
+
+Supported sensor models include:
+
+| Model | Description |
+| ----- | ----------- |
+| [`fake`](fake) | A model used for testing, with no physical hardware. |
+| [`ultrasonic`](ultrasonic) | [HC-S204 ultrasonic distance sensor](https://www.sparkfun.com/products/15569) |
+| [`bme280`](bme280) | [BME280 environmental sensor](https://www.adafruit.com/product/2652) |
+| [`ds18b20`](ds18b20) | [DallasTemperature DS18B20 digital temperature sensor](https://www.adafruit.com/product/381) |
+| [`power_ina219`](power_ina219) | [INA219 current sensor](https://www.amazon.com/dp/B07QJW6L4C) |
+| [`renogy`](renogy)| [Renogy battery temperature sensor](https://www.amazon.com/Renogy-Battery-Temperature-Sensor-Controllers/dp/B07WMMJFWY) |
+| [`sensirion-sht3xd`](sensirion-sht3xd) | [Sensirion SHT3x-DIS temperature and humidity sensor](https://www.adafruit.com/product/2857) |
+
+You can implement a model of sensor that is not natively supported by Viam by [creating and registering your own model of a sensor](/program/extend/modular-resources/).
+This allows you to have the same access and control of the sensor through Viam as you would if it was a built-in model.
+
+See [the Viam Labs GitHub](https://github.com/viam-labs/wifi-sensor) for an example of using the Viam Go SDK to implement a [WiFi strength sensor](https://github.com/viam-labs/wifi-sensor/blob/main/linuxwifi/linuxwifi.go), and [the Viam Robotics GitHub](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/module) for an example of creating a custom component module with the Viam Python SDK.
```

Comment:
How about

```suggestion
For an example of creating a custom component, see a [WiFi strength sensor built with the Viam Go SDK](https://github.com/viam-labs/wifi-sensor/blob/main/linuxwifi/linuxwifi.go) or the same WiFI strength sensor [implemented with the Viam Python SDK](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/module).
```

Reply:
making some changes here but unfortunately it's not quite the same example!